templates/project/.github/settings.yml.twig

```diff
@@ -0,0 +1,4 @@
+repository:
```

Comment:
I thought you would extend our dev-kit to do the stuff probot is doing.

Adding a config for an other config tool does not make any sense to me.

Reply:
It's the same strategy than github actions, linters, ...

For instance, we didn't extends our dev-kit to do the stuff that github actions are doing.
We did extend our dev-kit to have a only one place where we add the github actions and it's dispatched in every projects.

Instead of opening 20 PR, you'll have just to open one on Webkit, and the dispatch will do the job for you.

Plus it allows to synchronize the packagist description/keywords and the github description/topics like I proposed.
It can also automatically add `[Abandonned]` as soon as you update the composer.json.
We could also synchronize the packagist homepage and the github homepage.